src/components/container/LabResultsBloodType/LabResultsBloodType.tsx

```diff
@@ -0,0 +1,149 @@
+import MyDataHelps from "@careevolution/mydatahelps-js";
+import React, { useEffect, useState } from "react";
+import { Action, LoadingIndicator } from "../../presentational";
+import "./LabResultsBloodType.css";
+import "../HealthPreviewSection/HealthPreviewSection.css"
+import language from "../../../helpers/language";
+import { bloodTypeLabs, singleLabs, manyLabs } from "./LabResultsBloodType.previewdata";
+import { faInfoCircle } from "@fortawesome/free-solid-svg-icons";
+import iconBloodType from "./icon-bloodtype.svg";
+
+export interface LabResultsBloodTypeProps {
+    previewState?: "BloodTypeLabs" | "SingleLabs" | "ManyLabs" | "NoData"
+    summaryResultsOnly?: true | false
+    maximumResults?: number
+    onClick?: any
+    externalInfoLink?: true | false
+    externalInfoLinkDefault?: string
+    externalInfoLinkSpanish?: string
+    innerRef?: React.Ref<HTMLDivElement>
+}
+
+export default function (props: LabResultsBloodTypeProps) {
+    const [model, setModel] = useState<any>(null);
+
+    function getLabResultsSummary() {
+        if (props.previewState == "BloodTypeLabs") {
+            setModel(bloodTypeLabs);
+            return;
+        }
+        if (props.previewState == "SingleLabs") {
+            setModel(singleLabs);
+            return;
+        }
+        if (props.previewState == "ManyLabs") {
+            setModel(manyLabs);
+            return;
+        }
+        if (props.previewState == "NoData") {
+            setModel({
+                bloodTypeLabs: []
+            });
+            return;
+        }
+
+        var endpoint = 'HealthAndWellnessApi.LabResultsBloodType';
```

Comment:
Can you add a corresponding PR for this...curious what the returned model looks like